crossbar/router/longpoll.py

```diff
@@ -289,6 +289,11 @@ def __init__(self, parent, transport_details):
         self._serializer = transport_details['serializer']
         self._session = None
 
+        if(hasattr(self._parent, '_auth')):
+            # Create a config entry so that the auth settings are picked up by the session
```

Comment:
don't need `()` around the if; just `if hasattr(..)`.

Perhaps the comment should say when this would be true? What sort of parent will have `._auth`? What sort of parent *won't* have this attribute?

Is there a reason that a "config" object (or "auth=" kwarg) isn't simply passed into this `__init__` directly?

Reply:
Good questions. It's been a couple months since I looked at this, so I'll have to swing back around in the near future and refresh my memory on what I was thinking when I made these changes :)

Reply:
Okay :+1:

I've not sure if I've ever delved directly into the longpoll things, but am familiar with auth and the code "in general" -- so feel free to ping me on #autobahn in Freenode or Crossbar.io Gitter: https://gitter.im/crossbario/public if you have questions etc.

Reply:
(BTW, if the way things are organized currently seems "weird" -- it might be because it's weird and needs some refactoring! So please do get in touch, especially if you think there's a better way -- we're not afraid to refactor if it makes the code nicer/easier to read!)